lib/ses-to-cloudwatch/plugin.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace HM\AWS_SES_WP_Mail\CloudWatch;
+
+use Exception;
+use Aws\CloudWatchLogs\CloudWatchLogsClient;
+use Aws\Exception\AwsException;
+
+add_action( 'aws_ses_wp_mail_ses_sent_message', __NAMESPACE__ . '\\on_sent_message', 10, 2 );
```

Comment:
I forget if it's PSR or what, but I know there's a standard that forbids files with side-effects. I'd align with that and say I'd like to see this added somewhere else so that this file can be `require`d without anything else happening.

Reply:
This is the main plugin file, it's "meant" to include the side-effects.